Padanie8/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Padanie8
+{
+    internal class Program
+    {
+        public static void xfunc(double x)
```

Comment:
Именуйте методы с большой буквы
```suggestion
        public static void Xfunc(double x)
```